components/src/utils/utils.ts

```diff
@@ -42,6 +42,7 @@ function setDefaultHeaders(): HeadersInit {
     'flinkey-API-Key': Globals.apiKey,
     'Content-Type': 'application/json',
     'Accept': 'application/json',
+    'Customer-ID': '1',
```

Comment:
Please don't commit such values. This should already be part of `Globals` class. Please use value from there.